internal/image/selector.go

```diff
@@ -64,6 +64,9 @@ type SelectorOptions struct {
 	// Creds holds optional credentials for authenticating to the image
 	// repository.
 	Creds *Credentials
+	// InsecureSkipVerify is an optional flag, that if set to true, will disable
+	// verification of the image repository's TLS certificate.
+	InsecureSkipVerify bool
```

Comment:
Nit: The field name isn't clear that _cert_ verification is the verification it skips. I'd also be ok with dropping the "insecure" prefix here because I see that as more of a scare word for _user facing_ things. (So it's good that it is in the corresponding field in the CRD.)

None of this is a hill I'd die on. :100:% dealer's choice.

Reply:
I changed it to `InsecureSkipTLSVerify` as this appears to be the common name in the code base.